mypy/build.py

```diff
@@ -294,7 +298,9 @@ def default_lib_path(data_dir: str,
     # Add fallback path that can be used if we have a broken installation.
     if sys.platform != 'win32':
         path.append('/usr/local/lib/mypy')
-
+    assert path, "Could not resolve typeshed subdirectories. If you are using MyPy " \
```

Comment:
I'd like to plead for starting at 2, since mypy still supports 3.2 (as the check target -- not to run mypy itself). It just avoids confusion for people who read the source. (Also it should probably be a DEFINED_CONSTANT somewhere.)